<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndDialog</class>
 <widget class="QDialog" name="EndDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deadlock Algorithm Simulation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/resources/Icon_White.png</normaloff>:/resources/Icon_White.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 61, 85);</string>
  </property>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>371</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(73, 81, 103);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="end_pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 5px;
	background-color: rgb(105, 111, 128);
	color: rgb(235, 235, 235);
	font: 18pt &quot;Futura Hv BT&quot;;
}
QPushButton::pressed {
    border-radius: 5px;
	background-color: rgb(43, 48, 63);
	font: 18pt &quot;Futura Hv BT&quot;;
	color: rgb(217, 217, 217);
}
QPushButton::hover {
	background-color: rgb(152, 161, 186);
}
</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 18pt &quot;Futura Hv BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>Test Completed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="restart_pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 5px;
	background-color: rgb(105, 111, 128);
	color: rgb(235, 235, 235);
	font: 18pt &quot;Futura Hv BT&quot;;
}
QPushButton::pressed {
    border-radius: 5px;
	background-color: rgb(43, 48, 63);
	font: 18pt &quot;Futura Hv BT&quot;;
	color: rgb(217, 217, 217);
}
QPushButton::hover {
	background-color: rgb(152, 161, 186);
}
</string>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_runtime_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Bk BT</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 12pt &quot;Futura Bk BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>0.00 s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="resources">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Bk BT</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 12pt &quot;Futura Bk BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>Resources:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="end_resources_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Bk BT</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 12pt &quot;Futura Bk BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="statistics_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 14pt &quot;Futura Hv BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avg_A_value_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Bk BT</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 12pt &quot;Futura Bk BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avg_A_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Bk BT</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 12pt &quot;Futura Bk BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>average Resource Time A:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avg_C_value_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Bk BT</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 12pt &quot;Futura Bk BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avg_B_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Bk BT</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 12pt &quot;Futura Bk BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>average Resource Time B:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avg_B_value_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Bk BT</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 12pt &quot;Futura Bk BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="avg_C_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Bk BT</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 12pt &quot;Futura Bk BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>average Resource Time C:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="resources_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Bk BT</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
font: 12pt &quot;Futura Bk BT&quot;;
background-color: none;</string>
   </property>
   <property name="text">
    <string>Runtime</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
